week-4-practise/level-1/exercise-1.js

```diff
@@ -41,4 +41,13 @@ var writers = [
 
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
-*/
\ No newline at end of file
+*/
+writers.forEach(function (writer) {
+  if (writer.alive) {
+    console.log(
+      `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${

```

Comment:
Brilliant to see you using ES6 features 👏🏻 Definitely a good place to use string literals.